vacation-calendar/src/Code.js

```diff
@@ -0,0 +1,130 @@
+// Set the ID of the team calendar to add events to. You can find the calendar's
+// ID on the settings page.
+var TEAM_CALENDAR_ID = 'ENTER_TEAM_CALENDAR_ID_HERE';
+// Set the email address of the Google Group that contains everyone in the team.
+var GROUP_EMAIL = 'ENTER_GOOGLE_GROUP_EMAIL_HERE';
+
+var KEYWORDS = ['vacation', 'ooo', 'out of office', 'offline'];
```

Comment:
I have no idea which would be faster :) Don't think it matters too much, 900+ users is more than enough for the target use case. Just useful to know what the limits are. Plus it could make for a good article later on about how scaling :)

Reply:
SGTM, leaving it as-is for now.